clients/java/src/main/java/io/camunda/zeebe/client/impl/response/ProcessImpl.java

```diff
@@ -36,6 +36,25 @@ public ProcessImpl(final ProcessMetadata process) {
         process.getTenantId());
   }
 
+  /**
+   * A constructor that provides an instance with an empty String for a tenantId.
+   *
+   * <p>From version 8.3.0, the java client supports multi-tenancy for this command, which requires
+   * the <code>tenantId</code> property to be defined. This constructor is only intended for
+   * backwards compatibility in tests.
+   *
+   * @deprecated since 8.3.0, use {@link ProcessImpl#ProcessImpl(long processDefinitionKey, String
+   *     bpmnProcessId, int version, String resourceName, String tenantId)}
+   */
+  @Deprecated
+  public ProcessImpl(
+      final long processDefinitionKey,
+      final String bpmnProcessId,
+      final int version,
+      final String resourceName) {
+    this(processDefinitionKey, bpmnProcessId, version, resourceName, "");
+  }
```

Comment:
:+1: Then I will remove them. My rationalization was that these constructors might be used by the Java client users for their tests (similarly to how we use them in our tests).

Reply:
💭 That's of course totally possible. And it might be a good idea to deprecate them first. I just wanted to point out that it's not required from our side. The nice thing to do is to deprecate them first.